src/assets/javascript/components/buttom.js

```diff
@@ -0,0 +1,49 @@
+let btnReuslt = document.querySelector(".js-result");
+
+function buttom(textContent) {
+	
+let jsButton = document.createElement('button');
```

Comment:
You must return an html

Reply:
@Willian1661

Reply:
Try to add buttons before get elements from html 

https://stackoverflow.com/questions/64506259/how-to-insert-onclick-event-in-html-string

Reply:
Ok,i've already changed all the components for this ways and i it's a lot way easier 😅